src/Liquidator_NEW.sol

```diff
@@ -0,0 +1,75 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.19;
+
+import { LogExpMath } from "./libraries/LogExpMath.sol";
+import { IFactory } from "./interfaces/IFactory.sol";
+import { ERC20, SafeTransferLib } from "../lib/solmate/src/utils/SafeTransferLib.sol";
+import { IAccount_NEW } from "./interfaces/IAccount_NEW.sol";
+import { ILendingPool_NEW } from "./interfaces/ILendingPool_NEW.sol";
+import { Owned } from "lib/solmate/src/auth/Owned.sol";
+import { ILiquidator_NEW } from "./interfaces/ILiquidator_NEW.sol";
+import { RiskModule } from "lib/accounts-v2/src/RiskModule.sol";
+
+contract Liquidator_NEW is Owned {
+    using SafeTransferLib for ERC20;
+
+    /* //////////////////////////////////////////////////////////////
+                                STORAGE
+    ////////////////////////////////////////////////////////////// */
+
+    uint256 locked;
+
+    /* //////////////////////////////////////////////////////////////
+                                EVENTS
+    ////////////////////////////////////////////////////////////// */
+
+    /* //////////////////////////////////////////////////////////////
+                                CONSTRUCTOR
+    ////////////////////////////////////////////////////////////// */
+
+    constructor() Owned(msg.sender) {
+        locked = 1;
+    }
+
+    /* //////////////////////////////////////////////////////////////
+                                MODIFIERS
+    ////////////////////////////////////////////////////////////// */
+
+    modifier nonReentrant() {
```

Comment:
Why do we need a nonReentrant lock on the Liquidator?

Reply:
For liquidateAccount is used now, it will be deleted in later steps if its not necessary